app/Repositories/LocationRepository.php

```diff
@@ -92,7 +77,7 @@ public function getDatesOfFilter($filter)
         return $array_dates;
     }
 
-    public function analystic($full_time_seat, $shiftData, $filter, $title, $className)
+    public function analystic($full_time_seat, $shiftData, $filter, $title, $className, $total_seat)
     {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Variable "total_seat" is not in valid camel caps format